doc/sphinxext/sphinx_gallery/_static/gallery.css

```diff
@@ -190,25 +190,3 @@ p.sphx-glr-signature a.reference.external {
   margin-left: auto;
   display: table;
 }
-
-a.sphx-glr-code-links:hover{
-    text-decoration: none;
-}
-
-a.sphx-glr-code-links[tooltip]:hover:before{
```

Comment:
Why was this removed? The tooltips are now broken.

I am sorry, I can't review every PR to sphinx-gallery, but that's a bug. What was the logic?